ngraph/frontend/onnx/frontend/src/frontend.cpp

```diff
@@ -49,3 +85,63 @@ std::shared_ptr<ngraph::Function> FrontEndONNX::decode(InputModel::Ptr model) co
     NGRAPH_CHECK(model_onnx != nullptr, "Invalid input model");
     return model_onnx->decode();
 }
+
+std::string FrontEndONNX::get_name() const {
+    return "onnx_experimental";
```

Comment:
Why "experimental"? 😄

Reply:
right, we can change it to "onnx", but onnx frontend is still skipped in mo.py